{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/forms/radio.jinja

```diff
@@ -0,0 +1,38 @@
+{% from 'forms/attrs.jinja' import form_attrs %}
+
+{% macro radio(name=widget.name, value=widget.value, optgroups=widget.optgroups) %}
+    <div class="radio py-4">
+        <div class="flex gap-4">
+            {% for group_name, group_choices, group_index in optgroups %}
+                {{ radio_optgroup(group_name, group_choices, group_index) }}
+            {% endfor %}
+        </div>
+    </div>
+{% endmacro %}
+
+{% macro radio_optgroup(name, choices, index) %}
+    {% for choice in choices %}
+        {% if choice.wrap_label %}
+            <label for="{{ choice.id }}">
```

Comment:
Getting an error here about the use of ID. Should this be choice.value instead?

Reply:
There are generated IDs for this, but it's in `choice.attrs.id` not `choice.id` - have fixed this now.